exercises/async-activity-completion/solution/workflow.py

```diff
@@ -0,0 +1,42 @@
+import asyncio
+from typing import List
+
+from temporalio import activity, workflow
+from temporalio.client import Client
+from datetime import timedelta
+
+class GreetingComposer:
```

Comment:
Would not recommend any activity code in the workflow file in these courses (even though we do for succinct samples)